scala/akkahttp/build.sbt

```diff
@@ -14,6 +14,7 @@ assemblyOption in assembly := (assemblyOption in assembly).value
 assemblyJarName in assembly := "server_scala_akkahttp"
 
 libraryDependencies ++= Seq(
-  "com.typesafe.akka" %% "akka-http" % "[10.0,10.1)",
-  "com.typesafe.akka" %% "akka-http-core" % "[10.0,10.1)"
+  "com.typesafe.akka" %% "akka-http" % "10.1.11",
+  // "com.typesafe.akka" %% "akka-http-core" % "10.1.11"
+  "com.typesafe.akka" %% "akka-stream" % "2.6.4"
```

Comment:
is this safe to specify the version _any_ `here`
I mean, for each release of `akka-stream` we should bump if we specify a version `here`

Reply:
You don't _need_ to upgrade for each new version of akka-stream. But you can. The choice (for akka-http) was made that you **need** to specify the version of akka-stream you want to use.